app/views/stream_entries/_detailed_status.html.haml

```diff
@@ -5,14 +5,15 @@
         = image_tag status.account.avatar.url(:original), width: 48, height: 48, alt: '', class: 'u-photo'
     %span.display-name
       %strong.p-name.emojify= display_name(status.account)
-      %span.p-nickname= acct(status.account)
+      %span= acct(status.account)
 
-  .status__content.e-content.p-name.emojify<
+  .status__content.p-name.emojify
```

Comment:
The line is missing `<` at the end. That removes extra whitespace from the output HTML, which *does* get picked up visually because we want to support ASCII art in toot text